spec/features/user_can_view_past_orders_spec.rb

```diff
@@ -64,37 +37,24 @@
   end
 
   scenario "authenticated user cannot see other users orders" do
-    user = User.create(first_name: "default",
-                       last_name: "user",
-                       username: "default_user",
-                       password: "default",
-                       role: 0 )
-    user_2 = User.create(first_name: "default",
-                       last_name: "user",
-                       username: "default_user_2",
-                       password: "default",
-                       role: 0 )
-    order = user.orders.create(status: "Paid",
-                               total_price: 12)
-    order_2 = user.orders.create(status: "Pending",
-                                 total_price: 24)
-    order_3 = user_2.orders.create(status: "Cancelled",
-                                   total_price: 36)
-
-    visit login_path
+    user_1 = create(:user)
+    user_2 = create(:user)
+    travesty = create(:travesty_with_items)
+    item_1, item_2, item_3, item_4 = travesty.items
+    order_1 = create(:order, user_id: user_1.id)
+    order_2 = create(:order, user_id: user_1.id)
+    order_3 = create(:order, user_id: user_2.id, status: "completed", total_price: 10000)
```

Comment:
Line is too long. [89/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).